static/img/spectrum-code.svg

```diff
@@ -0,0 +1,19 @@
+<svg id="icon" xmlns="http://www.w3.org/2000/svg">
```

Comment:
We can inline these svg files right in the react code. This will reduce additional requests as it will be included in the JS bundle.

Reply:
Good call. Inlined.